File: Examples2018/NotificationToast/app/src/main/java/course/examples/notification/toast/NotificationToastActivity.java
package course.examples.notification.toast;

import android.app.Activity;
import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

public class NotificationToastActivity extends Activity {
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main);

    }

    public void onClick(@SuppressWarnings("unused") View v) {
        Toast.makeText(getApplicationContext(), getString(R.string.youre_toast_string), Toast.LENGTH_LONG).show();
    }
}

File: ExamplesKotlin/NotificationToast/app/src/main/java/course/examples/notification/toast/NotificationToastActivity.kt
package course.examples.notification.toast

import android.app.Activity
import android.os.Bundle
import android.view.View
import android.widget.Toast

class NotificationToastActivity : Activity() {
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.main)

    }

    fun onClick(v: View) {
        Toast.makeText(
            applicationContext,
            getString(R.string.youre_toast_string),
            Toast.LENGTH_LONG
        ).show()
    }
}